classifier.py

```diff
@@ -202,42 +202,72 @@ def default_target_age_for_audience(cls, audience):
         return cls.nr(None, None)
 
     @classmethod
-    def default_audience_for_target_age(cls, target_age):
-        """The default audience for a given target age.
-
-        Inverse of default_target_age_for_audience.
-        """
-        if not target_age:
-            # We were not passed a NumericRange
+    def default_audience_for_target_age(cls, nr):
```

Comment:
There were three partial implementations of this code (which turns an age range into an audience) in classifier.py. I consolidated them here.